### PR TITLE
Cache 32bit agent

### PR DESCRIPTION
**Pull request by joshsamuelson**
Downloads and unzips the 32bit agent installer.
Allows classifying with 32bit pe-repo class while offline.

### CODE REVIEW

File: manifests/get_32bit_agent.pp [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+  $version_dir  = "${public_dir}/${version}"
+  $agent_dir    = "puppet-enterprise-${version}-el-${operatingsystemmajrelease}-i386-agent"
+  $agent_file   = "${agent_dir}.tar.gz"
+  $url          = "https://s3.amazonaws.com/pe-builds/released/${version}"
+
+  file { [$puppet_dir,$repo_dir,$public_dir,$version_dir]:
+    ensure => directory
+  }
+  staging::deploy { $agent_file:
+    source  => "${url}/${agent_file}",
```
Comment (casharma) [inline manifests/get_32bit_agent.pp]: https://s3.amazonaws.com/pe-builds/released/latest/puppet-enterprise-latest-el-6.6-i386-agent.tar.gz doesn't seem to work.

Comment (joshsamuelson) [inline manifests/get_32bit_agent.pp]: Ah, you're right... let me see if I can find another URL.  It works if you give it the version number

Comment (casharma) [inline manifests/get_32bit_agent.pp]: $agent_dir should not have ${version} in it, if $version = latest by default.

Comment (joshsamuelson) [inline manifests/get_32bit_agent.pp]: Ok, it looks like the latest thing doesn't work on the main URL either, so I'm just going to set the default to 3.7.2

**Comment by joshsamuelson** [comment]
Ok, should be ready to merge.

**Pull request merged**